doc/classes/PhysicsServer2D.xml

```diff
@@ -592,11 +592,12 @@
 			<param index="1" name="callable" type="Callable" />
 			<param index="2" name="userdata" type="Variant" default="null" />
 			<description>
-				Sets the function used to calculate physics for the body, if that body allows it (see [method body_set_omit_force_integration]).
-				The force integration function takes the following two parameters:
+				Sets the body's force integration callback function, which can be used e.g. to update the body's linear and angular velocity (for example, based on contact with other bodies).
```

Comment:
This is basically saying "for example" twice which is a bit odd.
```suggestion
				Sets the body's force integration callback function to [param callable]. This can be used to update the body's linear and angular velocity (for example, based on contact with other bodies).
```